scripts/lisk/start.sh

```diff
@@ -4,18 +4,32 @@ command -v shellcheck > /dev/null && shellcheck "$0"
 
 # A temporary dir created to make the docker-compose files from
 # https://github.com/LiskHQ/lisk/tree/v1.4.0/docker available.
-# Only used as long as this script runs
-TMP_DIR=$(mktemp -d "${TMPDIR:-/tmp}/lisk_start.XXXXXXXXX")
+TMP_DIR="${TMPDIR:-/tmp}/@iov"
+mkdir "$TMP_DIR" || true
 
+LISK_DIR="lisk"
 # can be a branch or tag
-LISK_BRANCH="v1.4.0"
+LISK_TAG="v1.4.0"
 # Use a version available on docker hub: https://hub.docker.com/r/lisk/core/tags/
 LISK_DOCKER_VERSION="1.4.0"
 
 (
   cd "$TMP_DIR"
   echo "Navigated to temp directory $(pwd)"
-  git clone --depth 1 --branch "$LISK_BRANCH" https://github.com/LiskHQ/lisk.git
+  if [ -d "$LISK_DIR" ]; then
+    cd "$LISK_DIR"
+    if ! git describe --tags --exact-match --match="$LISK_TAG"; then
```

Comment:
All git commands support `-C`, which allows you to change directory for the git command. E.g.

```
git -C ./iov-core describe --tags
v0.14.1-131-g9ce68af5
```

Using `-C "$LISK_DIR"` here allows you to remove 3 `cd`s and the whole `else` branch